velox/docs/functions/presto/regexp.rst

```diff
@@ -32,6 +32,22 @@ limited to 20 different expressions per instance and thread of execution.
         SELECT like('abc', '%b%'); -- true
         SELECT like('a_c', '%#_%', '#'); -- true
 
+Not all patterns require compilation of regular expressions. For the patterns without custom escape:
```

Comment:
There is existing text

```
Not all patterns require
    compilation of regular expressions. Patterns 'hello', 'hello%', '_hello__%',....
```

We need to update it to add the new pattern. Currently, you are adding new text as if existing text doesn't exist or doesn't talk about similar thing.

Current:

```
Patterns 'hello', 'hello%', '_hello__%',
    '%hello', '%__hello_', '%hello%', where
```

New (proposed):

```
Patterns 'hello', 'hello%', '_hello__%',
    '%hello', '%__hello_', '%hello%', '%hello%velox%' where
```